api/app/Models/TestEntity.php

```diff
@@ -39,6 +39,16 @@ class TestEntity extends BaseModel {
     ];
 
     /**
+     * Get the access route for the entity.
+     *
+     * @return string
+     */
+    public function entityRoute()
+    {
+        return '/test/entities';
+    }
+
+    /**
```

Comment:
Yea I actually placed the comment poorly - it should have been for `public static function fakeTestEntity`.

I was just thinking about the `_self` attribute, and I think the best option is to have the model factory be able to return the attribute without trying to insert it on `->create`. See what I am working on at https://github.com/spira/spira/commit/58d89e7ffb9fad9580d972f8c4f60c62b2db0cf7 - I'll use the `getSelfAttribute()` from the classname